### DISCUSSION TITLE
How I can see the type info

### DISCUSSION THREAD

**Discussion by dinduck** [mrcjkb/rustaceanvim#370] [Q&A] [answered]
![](https://user-images.githubusercontent.com/1711539/124515923-4504e800-dde9-11eb-8d58-d97945a1a785.gif)

where i can see  f and tuple's type, whitch option should i enable

**Comment by mrcjkb** [comment, marked as answer]
[README.md/where-are-inlay-hints--type-hints](https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#where-are-inlay-hints--type-hints)
